README.md

```diff
@@ -85,3 +85,42 @@ steps:
 | `github-token` | ❌       | `github.token` | The GitHub token used to merge the pull-request     |
 | `command`      | ❌       | `merge`        | The command to pass to Dependabot                   |
 | `approve`      | ❌       | `true`         | Auto-approve pull-requests                          |
+
+### Configuration file syntax
+
+Using the configuration file `.github/auto-merge.yml`, you have the option to provide a more fine-grained configuration. The following example configuration file merges
+
+* minor development dependency updates
+* patch production dependency updates
+* minor security-critical production dependency updates
+
+```yml
+automerged_updates:
```

Comment:
the top-level key here seems redundant.

can simply get straight to listing `match` array

Reply:
I wasn't sure if that is possible, I'll change it

Reply:
I guess depends on the js lib ...